src/components/aktivitetskrav/vurdering/VurderAktivitetskravModal.tsx

```diff
@@ -27,13 +31,24 @@ export const VurderAktivitetskravModal = ({
   modalType,
   aktivitetskravUuid,
 }: VurderAktivitetskravModalProps) => {
+  const header = (): string => {
+    switch (modalType) {
+      case "AVVENT":
+        return texts.header.avvent;
+      case "IKKE_AKTUELL":
+        return texts.header.ikkeAktuell;
+      case undefined:
+        return "";
+    }
+  };
+
   return (
     <Modal
       closeOnBackdropClick
       onClose={() => setModalOpen(false)}
       open={isOpen}
       aria-labelledby={texts.modalContentLabel}
-      header={{ heading: "" }}
+      header={{ heading: header() }}
     >
```

Comment:
Ser vi har brukt `Modal.Header` litt lenger nede her. Er det en av disse vi foretrekker? 😊 Det hadde jo gått an å conditionally rendre `<Modal.Header>…` seksjon om `header()` ikke er `undefined` og returnere `null` om den er det.

Reply:
Går dette på [denne](https://github.com/navikt/syfomodiaperson/pull/1095#discussion_r1400697942)? Kan man ikke ha lukk knapp om man ikke har heading? 😊

Reply:
Kommer vel an på hvordan vi vil at header skal se ut. `<Modal.Header>` er når man ønsker noe mer custom styling enn det `header`-prop gir. Jeg har brukt førstnevnte lenger ned her for å forsøke å bevare stylingen vi hadde fra før.

Reply:
Stemmer at man må ha heading for å få lukk-knapp automagisk. Se https://aksel.nav.no/komponenter/core/modal.

Reply:
Synes oftest bare det er cleanest å ha children enn props😊 Men skjønner man mister lukk-knapp om man ikke rendrer `<Modal.Header` eller har heading property. Bare la det være slik det er nå da 🙌